examples/nanocoap_tool/Makefile

```diff
@@ -0,0 +1,47 @@
+# name of your application
+APPLICATION = nanocoap_tool
```

Comment:
Maybe we can rename that to something like "riot_as_cli_tool/"? If I imagine someone new to look at examples/, the name "nanocoap_tool" transports a very different intend compared to what is actually showcased in this example. What do you think?